ui/src/views/kubernetes/index.vue

```diff
@@ -131,13 +125,8 @@ const handleDragEnd = (_el: HTMLElement, newWidth: number) => {
   });
 };
 
-const handleUpdateWaterMarkParams = (content: string, rotate: number, width: number, height: number, fontSze: number, color: string) => {
+const handleUpdateWaterMarkContent = (content: string) => {
   waterMarkContent.value = content;
-  waterMarkRotate.value = rotate;
-  waterMarkWidth.value = width;
-  waterMarkHeight.value = height;
-  waterMarkFontSize.value = fontSze;
-  waterMarkColor.value = color;
 };
 
 onMounted(() => {
```

Comment:
None of the provided code appears to be part of a project or application; consequently, it cannot be checked for specific differences. For an accurate review, one would need actual source codes within an existing context.